addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateInfinityModel(model, infinityModel) {
+    model.pushObjects(infinityModel.get('content'));
+  },
+
```

Comment:
I'm still finding this confusing.  the `infinityModel` should always refer to the route's model, ie, the model stored at `_modelPath`.

I think we should do

```js
updateInfinityModel(collection, infinityModel) {
    if (!infinityModel) {
      infinityModel = this.get('_modelPath');
    }
    return infinityModel.pushObjects(collection.get('content'));
}
```

This would imply we'd want to change the server response in the promise above from `infinityModel` to `collection`, and then line 191 to: `this.updateInfinityModel(collection, model);`

I think the wording `infinityModel` should canonically always refer to the object at `route.get('_modelPath')` and nothing else.

This also means that developers can also use this function like so:

```js
actions: {
  pushHughIntoInfinityModel() [
    var updatedInfinityModel = this.updateInfinityModel([
      { id: 1, name: "Hugh Francis" }
    ]);
    console.log(updatedInfinityModel);
  }
}
```

...which is something some other devs have asked about.